packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -723,39 +729,18 @@ export const RemoteServerFactory = {
                     // purpose so we don't want to overwrite the header for those calls.
                     if (incomingRequest.path?.match(options.applySLASPrivateClientToEndpoints)) {
                         proxyRequest.setHeader('Authorization', `Basic ${encodedSlasCredentials}`)
+                    } else if (!incomingRequest.path?.match(options.slasApiPath)) {
+                        const message = `Request to ${incomingRequest.path} is not allowed through the SLAS Private Client Proxy`
+                        logger.error(message)
+                        return res.status(403).json({
+                            message: message
+                        })
                     }
 
                     // /oauth2/trusted-agent/token endpoint requires a different auth header
                     if (incomingRequest.path?.match(/\/oauth2\/trusted-agent\/token/)) {
                         proxyRequest.setHeader('_sfdc_client_auth', encodedSlasCredentials)
                     }
-                },
```

Comment:
Does this mean we now throw the error and do not log?

Reply:
Yes, removing this block allows SLAS errors to get through.

What this block was previously doing is intercepting any requests where the response from SLAS is HTTP 400 or greater and instead logging the same message.

I experimented with other wats of intercepting the response here and printing the message but that interferes with other built in error logging that we have (see `errorHandlerMiddleware in build-remote-server.js`)

Based on manual testing, with this block gone SLAS errors that get through the proxy are handled by code in commerce-sdk-react.